heartbeat/config/config.go

```diff
@@ -33,6 +39,7 @@ type Config struct {
 	Scheduler      Scheduler            `config:"scheduler"`
 	Autodiscover   *autodiscover.Config `config:"autodiscover"`
 	Jobs           map[string]*JobLimit `config:"jobs"`
+	Location       *LocationWithID      `config:"location"`
```

Comment:
Should we also change this to `RunFrom`

Reply:
Yeah, I was thinking this was a good top level name, but now I think you're right, consistency is more important.